### ISSUE TITLE
Clickable optgroup label doesn't trigger change event

### ISSUE THREAD

**Issue by CodeBrauer** [davidstutz/bootstrap-multiselect#799]
Using `enableClickableOptGroups: true` on a multiple select works fine. But clicking an optgroup-label will not trigger an "change" event on the select.

So the value(s) of the `<select>` changes but no event listener will detect this change.

![screenshot gif](https://cl.ly/1n3Z2C2F200o/Screen%20Recording%202016-09-06%20at%2011.28%20vorm..gif)

Demo of this issue on https://jsfiddle.net/85pbc12a/1/

I created a very quick and dirty fix:

``` js
$('.multiselect-group-clickable').bind('click', function() {
  // not the best way to find the select...
  $(this).parentsUntil('.btn-group').parent().parent().find('select').trigger('change');
}
```

Demo "fixed" version: https://jsfiddle.net/36jw8cdm/1/

**Comment by CodeBrauer** [comment]
Maybe this would be just a small change, but no time currently to test this.

Just include this line: `$(element).trigger('change');` in [`updateOptGroups`](https://github.com/davidstutz/bootstrap-multiselect/blob/93485181947b7b1492a3a5e3551a7e051f3e8e0e/dist/js/bootstrap-multiselect.js#L1547-L1579)

**Comment by john-clarke** [comment]
I too have this exact problem.

**Comment by davidstutz** [comment]
Will have a look at the PR, this should get fixed.

**Comment by davidstutz** [comment]
Merged #813.
